app/views/casa_admin_mailer/deactivation.html.erb

```diff
@@ -1,17 +1,45 @@
-<meta itemprop="name" content="Confirm Email" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-<table width="100%" cellpadding="0" cellspacing="0" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-  <tr style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-    <td class="content-block" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; vertical-align: top; margin: 0; padding: 0 0 20px;" valign="top">
+<style>
+  #table {
+    width: 100%;
+    border-spacing: 0;
+    font-family: Helvetica,Arial,sans-serif;
+    box-sizing: border-box;
+    font-size: 120px;
+    margin: 0;
+  }
+
+    #table tbody tr {
+    font-family: Helvetica,Arial,sans-serif;
+    box-sizing: border-box;
+    font-size: 14px;
+    margin: 0;
+  }
+  
+  .content-block {
+    font-family: Helvetica,Arial,sans-serif;
+    box-sizing: border-box;
+    font-size: 14px;
+    vertical-align: top;
+    margin: 0;
+    padding: 0 0 20px;
+  }
+
+</style>
+
+<meta itemprop="name" content="Confirm Email">
+<table id="table">
```

Comment:
You don't need ids to refer to these elements using css. you can just do `table {`